src/SDL_image.sym

```diff
@@ -41,6 +41,8 @@ SDL3_image_0.0.0 {
     IMG_SavePNG_IO;
     IMG_SaveAVIF;
     IMG_SaveAVIF_IO;
+    IMG_SaveGIF;
+    IMG_SaveGIF_IO;
```

Comment:
These should be moved to the bottom. Symbols should be added in the order they were added to the library (after the 3.2.0 release)

Reply:
Umm, ehm, guilty your honor, I had inserted SaveTGA stuff myself :)

We can rearrange and fix the symbols file after all these subside, can we not?